app/routers/dayview.py

```diff
@@ -26,6 +26,41 @@ class DivAttributes:
     CLASS_SIZES = ("title-size-tiny", "title-size-xsmall", "title-size-small")
     LENGTH_SIZE_STEP = (30, 45, 90)
 
+    def _minutes_position(self, minutes: int) -> dict[str, int]:
```

Comment:
I'd consider a short description here :)
Also - for typing, don't you need to use Dict (and not dict)?

Reply:
Preferably, use `Dict` from `typing` (`dict` is allowed only from Python >= 3.9)